### PR TITLE
SevenTV.chatterino7 version 7.3.4

### PR DESCRIPTION
**Pull request by s-crypt**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25119)

### CODE REVIEW

File: manifests/s/SevenTV/chatterino7/7.3.4/SevenTV.chatterino7.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: SevenTV.chatterino7
+PackageVersion: 7.3.4
+PackageLocale: en-US
+Publisher: SevenTV
+PackageName: Chatterino7
+License: MIT License
+ShortDescription: Chatterino7 Installer
```
Comment (ItzLevvie) [inline manifests/s/SevenTV/chatterino7/7.3.4/SevenTV.chatterino7.locale.en-US.yaml:10]:
```
ShortDescription: Chatterino7 Installer
```
Please could you write a better description about the application?

File: manifests/s/SevenTV/chatterino7/7.3.4/SevenTV.chatterino7.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: SevenTV.chatterino7
+PackageVersion: 7.3.4
+PackageLocale: en-US
+Publisher: SevenTV
```
Comment (ItzLevvie) [inline manifests/s/SevenTV/chatterino7/7.3.4/SevenTV.chatterino7.locale.en-US.yaml:7]:
```suggestion
Publisher: zneix
```

File: manifests/s/SevenTV/chatterino7/7.3.4/SevenTV.chatterino7.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: SevenTV.chatterino7
+PackageVersion: 7.3.4
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/SevenTV/chatterino7/releases/download/v7.3.4/Chatterino.Installer.exe
+  InstallerSha256: 5D39D61CD7F020DF09620EB4EB11ACCC90927A63EB2BB11E47A82E53FEDAB8BD
```
Comment (ItzLevvie) [inline manifests/s/SevenTV/chatterino7/7.3.4/SevenTV.chatterino7.installer.yaml:10]:
```suggestion
  InstallerSha256: 5D39D61CD7F020DF09620EB4EB11ACCC90927A63EB2BB11E47A82E53FEDAB8BD
  ProductCode: "{C4FF06D3-2C99-4E9A-96C6-8E99A666B681}_is1"
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please suppress these prompts:
image
image

image

**Comment by s-crypt** [comment]
Closed because I could not find a way to suppress prompts

**Pull request closed without merging**